packages/link/src/providers/WagmiCoreProvider.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC, PropsWithChildren } from 'react'
+import { config } from '../utils/wagmiCore'
+import { WagmiProvider as WagmiProviderInternal } from 'wagmi'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+
+export const WagmiCoreProvider: FC<PropsWithChildren> = ({ children }) => {
```

Comment:
our link solutiuon should not depend on react, please don't use react or and tsx here